packages/frontend/app/components/bulk-new-users.js

```diff
@@ -169,7 +168,7 @@ export default class BulkNewUsersComponent extends Component {
     this.primaryCohortId = id;
   }
 
-  async existingUsernames() {
+  async getExistingUserNames() {
```

Comment:
Do we not want to be consistent with naming convention here? I see `UserName`, but a few lines down on 184 you're using `Username`.

Reply:
Damn it, I hate it when very good and valid points are made! Good catch. I will fix it.... tomorrow.

Reply:
Fixed in #8258